src/App.js

```diff
@@ -65,27 +66,48 @@ const util = {
     }
     return res.filter(v => !v.includes('/'));
   },
-  
+
   getRand(min, max) {
     min = Math.ceil(min);
     max = Math.floor(max);
     return Math.floor(Math.random() * (max - min + 1)) + min;
   },
-  
+
   getRandomElement(arr) {
     return arr[util.getRand(0, arr.length - 1)];
   },
-  
+
+  rotateElements(arr, k) {
+    const n = k % arr.length;
+    if (n === 0) return arr;
+    arr.unshift( ...arr.splice(-n));
+    return arr;
+  },
```

Comment:
Helper function for inverting chords. If the notes in the "voicing" array aren't in the correct order, accidentals are applied to the incorrect notes when rendered visually (although most other parts of the chord work)